### ISSUE TITLE
Parameters name hints in IntelliJ

### ISSUE THREAD

**Issue by Pacane** [dart-lang/sdk#31220]
I was wondering if there was a way to get the parameters name hints in IntelliJ like there is for other languages:

![hints](https://d3nmt5vlzunoa1.cloudfront.net/idea/files/2016/12/idea_2017_1_groovy_parameter_hints.png)

**Comment by devoncarew** [comment]
Its not clear to me that this would be a net win, in terms of value vs making the editor busy, giving that Dart has named params in the language.  Definitely a useful feature for languages without named params.

/cc @jwren @alexander-doroshko

**Comment by Pacane** [comment]
Well, I understand it's useful for languages without named parameters, but not all Dart APIs use named parameters (except probably flutter)...

**Comment by devoncarew** [comment]
Yup, totally. I'm just saying that the language provides some options to help here. I've found the feature very useful in Java, eps. for methods that take one or more booleans. In Dart (assuming I have control of the APIs), I'd likely prefer using named params for clarity.

**Comment by jwren** [comment]
@devoncarew I agree with you on the point of named parameters.  (In our implementation we should of course not put the names in for named parameters, set methods, etc.)

There is also a question here for me of having the support in Dart simply to have all the features that developers recognize as features that IntelliJ provides.  If developers have a strong impression that the Dart support isn't done/complete/mature because of this, then we should add it.

/cc @stevemessick @alexander-doroshko

**Comment by helin24** [comment]
Closing as a duplicate of https://github.com/dart-lang/sdk/issues/30479
